src/explorer.api/Models/NumericOverview.cs

```diff
@@ -0,0 +1,115 @@
+﻿namespace Explorer
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Diagnostics;
+    using System.Linq;
+    using System.Text.Json;
+    using System.Threading.Tasks;
+
+    using Aircloak.JsonApi;
+    using Aircloak.JsonApi.ResponseTypes;
+    using Explorer.Api.Models;
+
+    internal class NumericOverview : ColumnExplorer
+    {
+        internal NumericOverview(JsonApiSession apiSession, ExploreParams exploreParams)
+            : base(apiSession, exploreParams)
+        {
+        }
+
+        internal async IAsyncEnumerable<ExploreResult> Explore()
+        {
+            var queryParams = new QuerySpec(ExploreParams);
+
+            yield return new ExploreResult(ExplorationGuid, status: "waiting");
+
+            var queryResult = await ApiSession.Query<ResultRow>(
+                ExploreParams.DataSourceName,
+                queryParams.QueryStatement,
+                TimeSpan.FromMinutes(2));
```

Comment:
You're right, I realise I have been scattering some arbitrary constants around... All of this will need to be gathered in configuration or environment variables but I'm not quite sure the best way to do this in c#/ASP.net so I have been delaying it... 

Other things to go in config :
- the aircloak server root url
- the api key
- anything else?

Good point about the cancel query endpoint! I thought it seemed sensible to set a timeout but to better mirror the functionality of Aircloak a `/cancel` endpoint might be better.